exec/jdbc/src/main/java/org/apache/drill/jdbc/SqlTimeoutException.java

```diff
@@ -23,12 +23,17 @@
  * Indicates that an operation timed out. This is not an error; you can
  * retry the operation.
  */
-public class SqlTimeoutException
-    extends SQLException
-{
+public class SqlTimeoutException extends SQLException {
+
+  private static final long serialVersionUID = 2017_06_20L;
```

Comment:
Fair enough... will set it to a -1. 
I saw a similar assignment for _InvalidParameterSqlException_ , which was set as a timestamp, so I just tried to follow the convention.